#! /bin/sh /usr/share/dpatch/dpatch-run
## 96-build-athena-3d.dpatch by  <pgquiles@elpauer.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Always define BUILD_ATHENA3D, needed for the tests

@DPATCH@

diff -rupd ACE_wrappers.orig/m4/ace.m4 ACE_wrappers/m4/ace.m4
--- ACE_wrappers.orig/m4/ace.m4	2009-11-05 22:47:48.000000000 +0100
+++ ACE_wrappers/m4/ace.m4	2009-11-06 00:43:58.000000000 +0100
@@ -1393,6 +1393,7 @@ if test "$no_x" != yes; then
    AC_SUBST(ACE_XT_LIBS)
 fi
 AM_CONDITIONAL([BUILD_ATHENA], true)
+AM_CONDITIONAL([BUILD_ATHENA3D], true)
 AM_CONDITIONAL([BUILD_MOTIF], false)
 ])
 
